typescript/packages/plugins/modespray/README.md

```diff
@@ -0,0 +1,34 @@
+# Goat Plugin ModeSrpray 🐐 💛 - TypeScript
+
+## Installation
+
+```bash
+pnpm install @goat-sdk/plugin-modespray
+```
+
+## Description
+
+This plugin enables AI agents to interact with ModeSpray on Mode Network, allowing them to **Spray** assets in a single click
+
+## Usage
+
+```typescript
+import { modeSprayPlugin } from "@goat-sdk/plugin-modespray";
+
+const tools = await getOnChainTools({
+    wallet: viem(wallet),
+    plugins: [
+        modeSprayPlugin(),
```

Comment:
No need to name it Plugin. I would just name it `modeSpray` the import and where you use it already shows it's a plugin

Reply:
Ah just saw that the function is actually called modespray, so the docs are out of date prob :)

Reply:
updated!